subscription-manager.spec

```diff
@@ -1117,15 +1117,15 @@ gtk-update-icon-cache -f %{_datadir}/icons/hicolor &>/dev/null || :
   (jhnidek@redhat.com)
 - ENT-447 Create .desktop file that opens web page with our cockpit plugin
   (jhnidek@redhat.com)
-- ENT-481 service-level command & options now update intent metadata
+- ENT-481 service-level command & options now update syspurpose metadata
```

Comment:
Something I didn't think of before: should we be changing the commit summaries in the spec file?

Reply:
No, don't change the spec file's changelog.  In this case it's probably fine since the package has never been released, but in general people will be grouchy if you change the changelog.